src/product-entry.js

```diff
@@ -0,0 +1,22 @@
+import store from './data/store.js';
+
+const form = document.getElementById('book-form');
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    const formData = new FormData(form);
+    const product = {
+        code: formData.get('code'),
```

Comment:
I would add an image field because it's coded into your render books functionality and right now when you enter a new book, you get the img symbol and the alt text